app/src/views/GrantRegistryNewGrant.vue

```diff
@@ -75,22 +86,30 @@ function useNewGrant() {
   const { poll } = useDataStore();
 
   // Define form fields and parameters
-  const form = ref<{ owner: string; payee: string; metaPtr: string }>({
+  const form = ref<{ owner: string; payee: string; name: string; description: string }>({
     owner: '',
     payee: '',
-    metaPtr: '',
+    name: '',
+    description: '',
```

Comment:
Should we define a local `Form` type, since I imagine we'll be adding more fields later too?

Reply:
If it's only used once I don't think it's necessary, but would defer to you guys